Sources/DistributedActors/ClusterSystem.swift

```diff
@@ -928,9 +933,6 @@ extension ClusterSystem {
             _openExistential(watcher, do: doMakeLifecycleWatch)
         }
 
-        let behavior = InvocationBehavior.behavior(instance: Weak(actor))
-        let ref = self._spawnDistributedActor(behavior, identifiedBy: actor.id)
-        self._managedRefs[actor.id] = ref
```

Comment:
hmmm maybe still keep it around? I'll try to see why we'd get dead letters right now still.